daemon/logger/awslogs/cloudwatchlogs_test.go

```diff
@@ -691,3 +692,33 @@ func TestCollectBatchWithDuplicateTimestamps(t *testing.T) {
 		}
 	}
 }
+
+func TestCreateTagSuccess(t *testing.T) {
+	mockClient := newMockClient()
+	ctx := logger.Context{
+		ContainerName: "/test-container",
+		ContainerID:   "container-abcdefghijklmnopqrstuvwxyz01234567890",
+		Config:        map[string]string{"tag": "{{.Name}}/{{.ID}}"},
+	}
+	logStreamName, e := loggerutils.ParseLogTag(ctx, loggerutils.DefaultTemplate)
+	if e != nil {
+		t.Errorf("Error generating tag: %q", e)
+	}
+	stream := &logStream{
+		client:        mockClient,
+		logGroupName:  groupName,
+		logStreamName: logStreamName,
+	}
+	mockClient.createLogStreamResult <- &createLogStreamResult{}
+
+	err := stream.create()
+
+	if err != nil {
+		t.Errorf("Received unexpected err: %v\n", err)
+	}
+	argument := <-mockClient.createLogStreamArgument
+
+	if *argument.LogStreamName != "test-container/container-ab" {
```

Comment:
Got it.  It might be nicer to change the template used for the test to one that doesn't involve truncation (maybe `{{.FullID}}`) so it's more obvious what's going on?